infinispan-remote/src/test/resources/storedprocedures/simpleValueProcedure.js

```diff
@@ -0,0 +1,2 @@
+// mode=local,language=javascript
+param;
```

Comment:
Could  you add a new line at the end of the file, please?

We couldn't find a way to add the check in checkstyle but it's the way we commit code